logging/logging.go

```diff
@@ -17,15 +17,23 @@
 // control where log messages end up.
 package logging
 
-import "log"
+import (
+	"log"
+	"os"
+)
 
 // Verbosef is called to write verbose logs, such as when a new connection is
-// established correctly.
```

Comment:
Please fix these comments you deleted?

Reply:
@Carrotman42 sorry this took so long, I've pushed a fix :+1: